### ISSUE TITLE
Claiming namespace deemp and renaming

### ISSUE THREAD

**Issue by deemp** [EclipseFdn/open-vsx.org#1589]
Hello! I renamed myself on GitHub from `br4ch1st0chr0n3` to `deemp`. May I please have my namespace renamed from `br4ch1st0chr0n3` to `deemp`?
I made an issue about claiming the `br4ch1st0chr0n3` namespace here: https://github.com/EclipseFdn/open-vsx.org/issues/1202

**Comment by kineticsquid** [comment]
@deemp We need this enhancement first: https://github.com/eclipse/openvsx/issues/427.

**Comment by deemp** [comment]
Got it. Thanks!

**Comment by kineticsquid** [comment]
@deemp You should be good now.

**Comment by deemp** [comment]
Hi! I see this error when trying to search for the extension:

```
Request failed: GET https://openvsxorg.blob.core.windows.net/resources/deemp/purs-keybindings/0.0.6/avatar.png (The specified blob does not exist.)
```

And when I access the extension at
https://open-vsx.org/extension/deemp/purs-keybindings

I get

```
Extension Not Found: deemp.purs-keybindings
```

**Comment by kineticsquid** [comment]
Tracking this new issue in https://github.com/eclipse/openvsx/issues/690.

**Comment by kineticsquid** [comment]
I've reverted this namespace change to make the extension available again. Re-opening this until we get the namespace code updated.

**Comment by kineticsquid** [comment]
@amvanbaren https://github.com/eclipse/openvsx/issues/690 is complete. Are we ready to try this rename again? Is there any DB surgery required because of the previous failure?

**Comment by amvanbaren** [comment]
@kineticsquid Yes, you can try the rename again. If the revert succeeded then there's no DB surgery required.

**Comment by kineticsquid** [comment]
@deemp Can you confirm?

**Comment by deemp** [comment]
Renamed successfully! Thank you!
